tc/tc_util.c

```diff
@@ -190,39 +190,42 @@ static const struct rate_suffix {
 	{ NULL }
 };
 
-int parse_percent_rate(char *rate, const char *str, char *dev)
+int parse_percent_rate(char *rate, const char *str, const char *dev)
 {
-	long max_rate_bits;
+	long mbit;
 	int ret, saved_errno;
 	double perc, rate_bits;
```

Comment:
Let's make `rate_bits` into `rate_mbit`, since that's the unit that's being read from the device's speed and then calculated.